Tests/scripts/check_base_branch_for_external_pr.py

```diff
@@ -0,0 +1,65 @@
+import requests
+import sys
+from Tests.test_utils import print_error, print_warning, print_color, LOG_COLORS
+
+
+def get_base_branch(pr_num):
+    """
+    Fetches the base branch name of PR num {pr_num}
+    :param pr_num: The PR number
+    :return: The name of the base branch if exists
+    """
+
+    # Disable insecure warnings
+    requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+    try:
+        url = 'https://api.github.com/repos/demisto/content/pulls/{}'.format(pr_num)
+        res = requests.get(url, verify=False)
+        if res.status_code != 200:
+            # If we didn't succeed to fetch the pr maybe it doesn't exist - then we don't want the build to fail
+            print_warning('Unable to fetch PR num {}'.format(pr_num))
+            return ''
+        response = res.json()
+        # Seems like GitHub API return a dict if it's only one PR, but it doesn't mentioned in their API
+        if response and isinstance(response, dict):
+            base = response.get('base', {})
+            base_branch = base.get('ref')
+            if base_branch:
+                return base_branch
+        # GitHub usually returns a list of PRs
```

Comment:
why would we get a list of PRs ?

Reply:
https://developer.github.com/v3/pulls/

look at the response showed here.